numpy/lib/_shape_base_impl.py

```diff
@@ -1241,16 +1244,29 @@ def tile(A, reps):
 
     >>> b = np.array([[1, 2], [3, 4]])
     >>> np.tile(b, 2)
+    array([[1, 2, 1, 2],
+           [3, 4, 3, 4]])
+    >>> np.tile(b, 2, axis=-1)
```

Comment:
Maybe I'm missing something, but it seems like every one of your examples is exactly the same as when no `axis` parameter is specified. That doesn't make it very clear what this actually does.

Reply:
I wanted to show what the equivalent call looks like without `axis`.  I thought that it might make it easier to understand its behavior.